### ISSUE TITLE
[Settings]: Screen Reader is announcing incorrect position number for tab item present in Profiles section.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11158]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Open Narrator using 'Win + Ctrl + Enter' keys.
Navigate to tab items present in Profile section and observe the issue.

**User Experience: -> Priority 3**
Screen Reader user are impacted here as incorrect position is defined for tab item controls which is confusing.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Screen Reader is announcing incorrect position number for tab item present below Profiles section..zip](https://github.com/microsoft/terminal/files/7116258/Screen.Reader.is.announcing.incorrect.position.number.for.tab.item.present.below.Profiles.section.zip)

### Expected Behavior

Screen Reader should announce correct position number for tab item present in Profiles section. If there are five tabs item, then screen Reader should announce 1 of 5 as position of first tab item and 5 of 5 for last/fifth tab item.

### Actual Behavior

Screen Reader is announcing incorrect position number for tab item present in Profiles section. Suppose there are five tabs items, but Screen Reader announce 1 of 4 as position of first tab item and for fifth tab item it announces as 4 of 4 as position.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue is fixed now.
